tests/Fixtures/inc/Engine/CriticalPath/AdminSubscriber/cpcssSectionIntegration.php

```diff
@@ -0,0 +1,288 @@
+<?php
+
+return [
+	'vfs_dir'   => 'wp-content/cache/critical-css/',
+
+	// Virtual filesystem structure.
+	'structure' => [
+		'wp-content' => [
+			'cache' => [
+				'critical-css' => [
+					'1' => [
+						'posts'         => [
+							'post-2.css' => '.p { color: red; }',
+						],
+					],
+				],
+			],
+			'plugins' => [
+				'wp-rocket' => [
+					'views' => [
+						'metabox' => [
+							'cpcss' => [
+								'container.php' => file_get_contents( WP_ROCKET_PLUGIN_ROOT . 'views/metabox/cpcss/container.php' ),
+								'generate.php' => file_get_contents( WP_ROCKET_PLUGIN_ROOT . 'views/metabox/cpcss/generate.php' ),
+								'regenerate.php' => file_get_contents( WP_ROCKET_PLUGIN_ROOT . 'views/metabox/cpcss/regenerate.php' ),
+							],
+						],
+					],
+				],
+			],
+		],
+	],
+
+	// Test Data
+	'test_data' => [
+		'testShouldDisplayGenerateTemplateDisabledWarningWhenOptionDIsabled' => [
+			'config' => [
+                'options' => [
+                    'async_css' => 0,
+                ],
+                'post' => [
+                    'post_status' => 'draft',
+                    'post_type'   => 'post',
+                    'ID'          => 1,
+                ],
+                'is_option_excluded' => true,
+			],
```

Comment:
Fix the formatting.

Reply:
https://github.com/wp-media/wp-rocket/pull/2573/commits/c60624efda4912cbabb55785c4e3cf06773fc497